src/components/Navigation/Pages/PageBrowse/ButtonsBar.js

```diff
@@ -20,8 +21,8 @@ export default class ButtonsBar extends Component {
             <span>&nbsp;Revert Changes</span>
           </Button>
         </ButtonWithTooltip>
-        <Button bsStyle="default" disabled={hasChanges} onClick={collapseAll}>
-          Collapse All
+        <Button bsStyle="default" onClick={toggleCollapseExpandAll}>
+          Toggle Collapse
```

Comment:
Should we really need an "expand all" method to expand all the shown definitions in the list? I'm not so sure about this.
@jeffmcaffer what do you think?

Reply:
Maybe I misinterpreted @jeffmcaffer's [comment](https://github.com/clearlydefined/website/issues/531#issuecomment-468717062). Happy to make it only collapse though I think the button should expand the components if you can collapse them